config/routes.rb

```diff
@@ -4,19 +4,24 @@
 
   # Sufia generated routes
   blacklight_for :catalog
-  devise_for :users
+
+  scope "/admin" do
+    devise_for :users
+  end
+
   Hydra::BatchEdit.add_routes(self)
-  # This must be the very last route in the file because it has a catch-all route for 404 errors.
-    # This behavior seems to show up only in production mode.
-  mount Sufia::Engine => '/'
+  # This must be the very last route in the file because it has a catch-all
+  # route for 404 errors.
+  # This behavior seems to show up only in production mode.
+  mount Sufia::Engine => "/admin"
 
   resources :searches
```

Comment:
Yes, we need it.  This is where the root route redirects to, and also all of the search form submissions.

We should maybe limit the routes, but that should wait until @doctorargus is done with the paging/ajax-ifying; there are some other routes that will need to be allowed.  I'm thinking that making this a resource is the wrong thing, as it isn't a RESTful thing any more.  But again, we should address that when @doctorargus is done.